Formula/sqlite.rb

```diff
@@ -32,14 +32,14 @@ class Sqlite < Formula
 
   resource "functions" do
     url "https://sqlite.org/contrib/download/extension-functions.c?get=25", :using => :nounzip
-    version "2010-01-06"
+    version "2010-02-06"
```

Comment:
The file itself didn't change. It most likely was a typo which remained undetected until now...